streams/service/src/main/java/com/hortonworks/streamline/streams/service/UDFCatalogResource.java

```diff
@@ -318,7 +293,8 @@ public Response downloadUdf(@PathParam("udfId") Long udfId, @Context SecurityCon
         throw EntityNotFoundException.byId(udfId.toString());
     }
 
-    private void processUdf(InputStream inputStream,
+    /* package-private (for unit-test) */
+    void processUdf(InputStream inputStream,
```

Comment:
Let's add annotation @VisibleForTesting instead if Guava is available for this module.